themes/README.md

```diff
@@ -1,6 +1,43 @@
-## Gatsby Themes and Starters
+<p align="center">
+  <a href="https://gatsbyjs.org">
+    <img alt="Gatsby" src="https://www.gatsbyjs.org/monogram.svg" width="60" />
+  </a>
+</p>
+<h1 align="center">
+   Gatsby Themes and Starters
+</h1>
+<h3 align="center">
+  ⚛️ 📄 :rocket:
+</h3>
+<p align="center">
+  <strong>Blazing fast modern site generator for React</strong><br>
+  Go beyond static sites: build blogs, e-commerce sites, full-blown apps, and more with Gatsby.
+</p>
+<p align="center">
+  <a href="https://github.com/gatsbyjs/gatsby/blob/master/LICENSE">
+    <img src="https://img.shields.io/badge/license-MIT-blue.svg" alt="Gatsby is released under the MIT license." />
+  </a>
+  <a href="https://circleci.com/gh/DSchau/starters">
+    <img src="https://circleci.com/gh/DSchau/starters.svg?style=shield" alt="Current CircleCI build status." />
+  </a>
+  <a href="https://www.npmjs.org/package/gatsby">
+    <img src="https://img.shields.io/npm/v/gatsby.svg" alt="Current npm package version." />
+  </a>
+  <a href="https://npmcharts.com/compare/gatsby?minimal=true">
+    <img src="https://img.shields.io/npm/dm/gatsby.svg" alt="Downloads per month on npm." />
+  </a>
+  <a href="https://gatsbyjs.org/docs/how-to-submit-a-pr/">
```

Comment:
this link is broken ...

Reply:
maybe this link is better then just a deep link to pr?

https://www.gatsbyjs.org/contributing/

Reply:
on homepage its:
```suggestion
  <a href="https://www.gatsbyjs.org/contributing/how-to-contribute/">
```

same broken link here:https://github.com/gatsbyjs/gatsby/blob/master/starters/README.md